docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -930,3 +930,50 @@ Then element located '$locator' disappears in '$timeout'
 When I click on a button with the name 'Toggle element visibility with delay'
 Then element located 'By.xpath(//div[@id='delayed'])' disappears in 'PT3S'
 ----
+
+== Tips & Tricks
+
+=== Validate url host component
+
+To validate host component of current page URL the following expression can be used:
+
+.Validate host component of current page URL
+[source,gherkin]
+----
+Then `#{extractHostFromUrl(${current-page-url})}` is equal to `$expected-page-host`
```

Comment:
note really user friendly, I guess it would be better to make the example as following (just snippet)
Given I am on page with url `https://dwqeqwe.com/qweqe`
Then `#{extractHostFromUrl(${current-page-url})}` is equal to `dwqeqwe.com`

Reply:
Added usage example below at each section

Reply:
this example should be removed, since better one is provided below